### PR TITLE
Feature/mycredits

### PR DESCRIPTION
**Pull request by kevinch**
#### What is the purpose of this pull request?
Transfer my credits to the Render.

#### Notes
- This is still work in progress. Please disconsider hard coded i18n.
- `getUserProfile` and `fetchStatements` do have each two lines of commented code for development purposes
- `react/my-credits.js` container will call other containers
- `myCreditsReducer` do have some missing cases.

#### Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: react/actions/ApiAddress.js [hunk 1]
```diff
@@ -10,8 +10,14 @@ export const putCheckingAccountTolerance = checkingAccountId =>
...
-  let endpoint = `${CA_BASE}?from=${data.from}&to=${data.to}`
-  if (data.queryString) {
+  let endpoint = `${CA_BASE}?`
+  if (data && data.from) {
+    endpoint += `from=${data.from}`
+  }
+  if (data && data.to) {
+    endpoint += `&to=${data.to}`
+  }
+  if (data && data.queryString) {
```
Comment (alcar) [inline react/actions/ApiAddress.js]: I believe it'd be better to check for data only once.

File: react/actions/mycredits/index.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+import { isNil, path } from 'ramda'
+
+const cookie = require('cookie-dough')()
+
+export const getAutCookie = account => {
+  var scopedCookie = cookie.get(`VtexIdclientAutCookie_${account}`)
+  if (!isNil(scopedCookie)) {
+    return scopedCookie
+  }
+  return cookie.get('VtexIdclientAutCookie')
```
Comment (alcar) [inline react/actions/mycredits/index.js]: Directly accessing the VTEX ID cookie won't be allowed in a somewhat near future. Wouldn't it be better to build the logic around getting the user email from `/api/vtexid/pub/authenticated/user` without using tokens? (Perhaps the same thing [here](https://github.com/vtex-apps/credit-control/pull/30/files#diff-4aa4ad86fdea30351b161af6924322cfR38))

File: react/actions/ApiAddress.js (hunk 1 above)
Comment (kevinch) [reply react/actions/ApiAddress.js]: True. I have an issue opened to better deal with this url building problem (https://github.com/vtex-apps/credit-control/issues/29).

File: react/actions/mycredits/index.js (hunk 2 above)
Comment (kevinch) [reply react/actions/mycredits/index.js]: If that's a possibility i'll gladly update.

**Pull request merged**